index.html

```diff
@@ -68,5 +68,260 @@ <h1>Hey There.<br />I'm Oluwaseyi.</h1>
+        <!-- ===== End of project card two ===== -->
+        <!-- ===== Project card -->
+        <section class="card">
+          <div class="cardImg">
+            <div>
+              <img
+                src="images\Img Placeholder (1).png"
+                alt="Card background image"
+                id="bgCard"
+              />
+              <button class="card2Btn">See project</button>
+            </div>
+
+            <div class="textCard">
+              <h4>Profesional Art<br />Printing Data</h4>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. Has been the industry's standard.
+              </p>
+              <ul class="projectLang">
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+          </div>
+        </section>
+        <!-- ===== End of project card two ===== -->
+        <!-- ===== Project card -->
+        <section class="card">
+          <div class="cardImg">
+            <div>
+              <img
+                src="images\Img Placeholder (1).png"
+                alt="Card background image"
+                id="bgCard"
+              />
+              <button class="card2Btn">See project</button>
+            </div>
+
+            <div class="textCard">
+              <h4>Profesional Art<br />Printing Data</h4>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. Has been the industry's standard.
+              </p>
+              <ul class="projectLang">
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+          </div>
+        </section>
+        <!-- ===== End of project card two ===== -->
+      </section>
+    </section>
+    <!-- ===== About Section -->
+    <section class="aboutpage">
+      <div class="aboutMe">
+        <h4>About me</h4>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <button>Get my resume</button>
+        <div id="aboutBorder"></div>
+      </div>
+    </section>
+    <section class="padding">
+      <section class="accord">
+        <div class="accordImg">
+          <img src="images\Rectangle 55.png" alt="" />
+          <h5>Language</h5>
+        </div>
+        <ul class="aboutList">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </section>
+      <section class="accord">
+        <div class="accordImg">
+          <img src="images\Rectangle 56.png" alt="" />
+          <h5>Frameworks</h5>
+        </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`